src/animations/SlideInOut.tsx

```diff
@@ -0,0 +1,29 @@
+import { ReactElement } from "react";
+import { motion } from "framer-motion";
+
+type Props = {
+  distanceInPx: number;
+  isActive: boolean;
+  children: ReactElement | ReactElement[];
+};
+
+export default function SlideInOut({
+  distanceInPx,
```

Comment:
What is this magic number? :D I think we need some comments here about how to use this.